llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -16473,9 +16484,12 @@ bool AArch64TargetLowering::lowerInterleaveIntrinsicToStore(
       R = Builder.CreateExtractVector(StTy, II->getOperand(1), Idx);
     }
 
-    if (UseScalable)
-      Builder.CreateCall(StNFunc, {L, R, Pred, Address});
-    else
+    if (UseScalable) {
+      SmallVector<Value *> Args(LeafNodes);
```

Comment:
You can use a plain C array here or you can pass a braced list to the `CreateCall`. You don't need the resizing capabilities of a `Smallvector`.